domains/is-cool.dev.is-cool.dev.json

```diff
@@ -0,0 +1,14 @@
+{
+    "domain": "is-cool.dev",
+    "subdomain": "is-cool.dev",
+
+    "owner": {
+        "email": "mustafayevalamiye7@gmail.com"
+    },
+
+    "record": {
+        "TXT": ["txt"]
```

Comment:
TXT record alone doesn't do anything.

Reply:
Brother, I have another domain name, I have already done what was written, what else are you waiting for me to do?